Sources/Widgets/Representations/LineGlyphRepresentation/example/index.js

```diff
@@ -0,0 +1,68 @@
+import '@kitware/vtk.js/favicon';
+
+// Load the rendering pieces we want to use (for both WebGL and WebGPU)
+import '@kitware/vtk.js/Rendering/Profiles/All';
+
+import vtkFullScreenRenderWindow from '@kitware/vtk.js/Rendering/Misc/FullScreenRenderWindow';
+import vtkStateBuilder from '@kitware/vtk.js/Widgets/Core/StateBuilder';
+
+import vtkSphereHandleRepresentation from '@kitware/vtk.js/Widgets/Representations/SphereHandleRepresentation';
+import vtkLineGlyphRepresentation from '@kitware/vtk.js/Widgets/Representations/LineGlyphRepresentation';
+
+// ----------------------------------------------------------------------------
+// Standard rendering code setup
+// ----------------------------------------------------------------------------
+
+const fullScreenRenderer = vtkFullScreenRenderWindow.newInstance({
+  background: [0, 0, 0],
+});
+const renderer = fullScreenRenderer.getRenderer();
+const renderWindow = fullScreenRenderer.getRenderWindow();
+
+// -----------------------------------------------------------
+// State
+// -----------------------------------------------------------
+
+const compositeState = vtkStateBuilder
+  .createBuilder()
+  .addDynamicMixinState({
+    labels: ['handles'],
+    mixins: ['origin', 'scale1'],
+    name: 'handle',
+  })
+  .build();
+
+const z = -50;
+const points = [
+  [0, 0, z],
+  [5, -5, z],
+  [15, 5, z],
+  [5, 10, z],
+];
+points.forEach((point) => {
+  const handle = compositeState.addHandle();
+  handle.setOrigin(point);
+  handle.setScale1(1);
+});
+
+// -----------------------------------------------------------
+// Representation
+// -----------------------------------------------------------
+
+const widgetRep = vtkLineGlyphRepresentation.newInstance({
+  scaleInPixels: false,
+  lineThickness: 0.5, // radius of the cylinder
+});
+widgetRep.setInputData(compositeState);
+widgetRep.setLabels(['handles']);
+widgetRep.getActors().forEach(renderer.addActor);
```

Comment:
a representation is a prop, therefore you shouldn't have to manually add all the actors of the representation to the renderer. Adding the representation to the renderer should be enough...

Reply:
When i change line 58 to `renderer.addActor(widgetRep);` getting runtime error
```
index.js:339 Uncaught TypeError: prop.getBounds is not a function
    at publicAPI.computeVisiblePropBounds (index.js:339:29)
```
Guess prop is not quite an actor?

Reply:
fair enough.

I guess a representation could be considered as an actor, but there must have been a good reason why it is just a prop (the widget view is however an actor with nested props being representations).

WDYT @floryst ?

Reply:
I guess because representations could be 2D as well. This is probably fine for the examples, as representations are not typically used directly.